framework/integration/config_map.go

```diff
@@ -7,7 +7,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func createConfigMap(configMap *corev1.ConfigMap) (*corev1.ConfigMap, error) {
+func CreateConfigMap(configMap *corev1.ConfigMap, namespace string) (*corev1.ConfigMap, error) {
```

Comment:
`namespace` needs to be supplied now, I want to reuse these functions for another testcase. 

I made these functions public to access them from the test package, I might move this into another package to get more alignment with `aws-operator` - WDYT?

Reply:
Separate package sounds good to me to not import the root package.

Reply:
Can't you take the namespace from `configMap`?

Reply:
Yes, I totally forgot that - changed @kopiczko 

I moved the `config_map.go` and `init.go` to a package called `client`. I simply couldn't think of a better name as they both use and initialize the k8s client. Open to other suggestions.